### PR TITLE
Niel breen zaps terrorists - WIP

### PR DESCRIPTION
**Pull request by CarlJMcGee**

### CODE REVIEW

File: NielBreenZapsTerrorists/NewZap/.gitattributes [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Normalize EOL for all files that Git considers text files.
+* text=auto eol=lf
```
Comment (EverybodyMakesFsteaks) [inline NielBreenZapsTerrorists/NewZap/.gitattributes:2]: Wonder what this does? Not really, but eh.

File: NielBreenZapsTerrorists/NewZap/Art/Breen1.png
Comment (EverybodyMakesFsteaks) [inline NielBreenZapsTerrorists/NewZap/Art/Breen1.png:1]: This picture belongs in a museum.

**Review by EverybodyMakesFsteaks — COMMENTED** [review]
Thus spoke Dr. Brinza

**Review by CarlJMcGee — COMMENTED** [review]
review

**Comment by EverybodyMakesFsteaks** [comment]
Note to self: The following bugs still exist in the Zap game that will need to be fixed:
~ Leadin is currently broken in two ways. 1) Text/pics are not becoming visible. 2) Fails to load Zap level. *Also, I feel the LeadIn is a little too long. I'll make a new one for the Shrek game, and if I like it better, will import back for consistency.
~Not a big deal, but while I still can't find how to reproduce bug, many testings have caused a victory after zapping 2 or 3 terrorists instead of all 4. Will need to identify and fix at some point.

**Pull request merged**